igvm_c/scripts/post_process.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2024 SUSE LLC
+#
+# Author: Roy Hopkins <roy.hopkins@suse.com>
+#
+# A script to post-process the header files generated with cbindgen
+# to rename items which could not be directly handled by the
+# cbindgen configuration files and annotations.
+set -e
+
+sed -i -e 's/INVALID = 0/IGVM_INVALID = 0/g' \
+        -e 's/RESERVED_DO_NOT_USE = /IGVM_RESERVED_DO_NOT_USE = /g' \
+        -e 's/RequiredMemoryFlags/IgvmRequiredMemoryFlags/g' \
+        -e 's/MemoryMapEntryFlags/IgvmMemoryMapEntryFlags/g' \
+        -e 's/u64_le/uint64_t/g' \
```

Comment:
is this saying to treat u64_le as uint64_t? isn't that not correct on BE platforms?

Reply:
Data types don't have any inherent endianness - the data held in them has endianness defined by whatever code is storing the value.

Reply:
To make the generated code more similar to the IGVM specification, which specifies endianness for fields using the `u64_le` and `u32_le` types I've now renamed these using cbindgen and added typedefs to the stdint types. It is still up to the consumer of the library to convert endianness if necessary, but at least it is more obvious from the data type.

```
IGVM_U64_LE = u64_le (uint64_t)
IGVM_U32_LE = u32_le (uint32_t)
```

Reply:
that makes sense thanks!